File: Products3/Form1.vb
Public Class Form1
  Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
    'TODO: This line of code loads data into the 'AdventureWorksDataSet.ProductCategory' table. You can move, or remove it, as needed.
    ProductCategoryTableAdapter1.Fill(AdventureWorksDataSet.ProductCategory)
    ProductModelTableAdapter1.Fill(AdventureWorksDataSet.ProductModel)
    ProductTableAdapter1.Fill(AdventureWorksDataSet.Product)
  End Sub
End Class

File: E09_Data1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace E09_Data1
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void productBindingNavigatorSaveItem_Click(object sender, EventArgs e)
        {
            this.Validate();
            this.productBindingSource.EndEdit();
            this.tableAdapterManager.UpdateAll(this.adventureWorksDataSet);

        }

        private void Form1_Load(object sender, EventArgs e)
        {
            // TODO: This line of code loads data into the 'adventureWorksDataSet.Product' table. You can move, or remove it, as needed.
            this.productTableAdapter.Fill(this.adventureWorksDataSet.Product);

        }
    }
}
